.github/workflows/set-milestone-from-base-branch.yml

```diff
@@ -38,23 +38,24 @@ jobs:
           URL_REGEX='(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)'
           UPLIFT_PR_REGEX='uplift\s+(?:of)?:?\s+\K(#\d+)'
           RESOLVE_ISSUE_REGEX="(?:resolv|fix|clos)(?:e|es|ed)?:?\s+\K(${URL_REGEX})"
+          verlte() { printf '%s\n%s' "$1" "$2" | sort -C -V; }
 
-          uplift_pr_body=$(gh pr view "$PR_NUMBER" -R "$REPO_NAME" --json body --jq '.body')
+          uplift_pr_body=$(gh pr view "${PR_NUMBER}" -R "${REPO_NAME}" --json body --jq .body)
```

Comment:
Why is this called `uplift_pr_body`? Isn't it the body of the current PR, which may, or may not be an uplift?